bin/server.py

```diff
@@ -0,0 +1,80 @@
+"""Main module cusdeb-helpik that stores handlers"""
+import re
+import sys
+import requests
+from aiohttp import web
+import aiohttp_cors
```

Comment:
The imports are not PEP8-compatible. See https://www.python.org/dev/peps/pep-0008/#imports.